windows/ReactNativeWebView/ReactWebView.idl

```diff
@@ -14,6 +14,7 @@ namespace ReactNativeWebView{
     runtimeclass ReactWebView2 : Windows.UI.Xaml.Controls.ContentPresenter{
         ReactWebView2(Microsoft.ReactNative.IReactContext context);
         void NavigateToHtml(String html);
+        void SetMessagingEnabled(Boolean enabled);
```

Comment:
prefer a property for this since it's just setting a bool:
```
Boolean MessagingEnabled { get; set; }
```

Reply:
This is the structure of the original WebView code. Seems like a good change to make though. I'll make a note to cleanup the original WebView as well in a PR after this.